org.eclipse.transformer.cli/src/main/resources/org/eclipse/transformer/jakarta/jakarta-renames.properties

```diff
@@ -72,35 +111,65 @@ javax.resource.spi=jakarta.resource.spi
 javax.resource.spi.endpoint=jakarta.resource.spi.endpoint
 javax.resource.spi.security=jakarta.resource.spi.security
 javax.resource.spi.work=jakarta.resource.spi.work
-
+javax.security.auth.message=jakarta.security.auth.message
+javax.security.auth.message.callback=jakarta.security.auth.message.callback
+javax.security.auth.message.config=jakarta.security.auth.message.config
+javax.security.auth.message.module=jakarta.security.auth.message.module
+javax.security.enterprise=jakarta.security.enterprise
+javax.security.enterprise.authentication=jakarta.security.enterprise.authentication
+javax.security.enterprise.authentication.mechanism=jakarta.security.enterprise.authentication.mechanism
+javax.security.enterprise.authentication.mechanism.http=jakarta.security.enterprise.authentication.mechanism.http
+javax.security.enterprise.credential=jakarta.security.enterprise.credential
+javax.security.enterprise.identitystore=jakarta.security.enterprise.identitystore
+javax.security.jacc=jakarta.security.jacc
 javax.servlet=jakarta.servlet
 javax.servlet.annotation=jakarta.servlet.annotation
 javax.servlet.descriptor=jakarta.servlet.descriptor
 javax.servlet.http=jakarta.servlet.http
-javax.servlet.resources=jakarta.servlet.resources
-
 javax.servlet.jsp=jakarta.servlet.jsp
 javax.servlet.jsp.el=jakarta.servlet.jsp.el
-javax.servlet.jsp.resources=jakarta.servlet.jsp.resources
-javax.servlet.jsp.tagext=jakarta.servlet.jsp.tagext
-
+javax.servlet.jsp.jstl=jakarta.servlet.jsp.jstl
 javax.servlet.jsp.jstl.core=jakarta.servlet.jsp.jstl.core
 javax.servlet.jsp.jstl.fmt=jakarta.servlet.jsp.jstl.fmt
 javax.servlet.jsp.jstl.sql=jakarta.servlet.jsp.jstl.sql
 javax.servlet.jsp.jstl.tlv=jakarta.servlet.jsp.jstl.tlv
-
+javax.servlet.jsp.resources=jakarta.servlet.jsp.resources
```

Comment:
This package does not seem to be part of the Jakarta EE 9 RC2 api.

Reply:
Is it possible to keep this? Tomcat's servlet api jar does include this package.

Reply:
More googling seems to point out this package is legit. Not sure why it is not in the RC2 api jar.

Reply:
I'll take a look and see if I can spot anything missing, or a specific reason its included in the Apache Tomcat version of servlet-api.

Reply:
This "resources" package is not part of the JSP API source:
https://github.com/eclipse-ee4j/jsp-api/tree/master/api/src/main/java/jakarta/servlet/jsp

At this point, I don't think this package should be included in the Transformer rules.